### PR TITLE
feature email send

### LINKED ISSUE #4: Generate Daily PDF Report
Export daily report in pdf with player's list and send as mail instead of storying in same directory.
Remove Xls or CSV from source directory

### PR DESCRIPTION
**Pull request by siddvish21**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,41 +1,51 @@
 # Minecraft Player List Tracker
 
-This script is designed to monitor changes to the player list on an Aternos Minecraft server, and record them in an Excel sheet. The script checks the player list every 10 seconds, and if a new player has joined or left the server, it updates the data in the Excel sheet.
+This script is designed to monitor changes to the player list on an Aternos Minecraft server, and record them in an Excel sheet and a CSV file. The script checks the player list every 10 seconds, and if a new player has joined or left the server, it updates the data in both the Excel sheet and the CSV file. Additionally, the script will send a PDF file of the day's player list as an email attachment at 8:00 PM every day.
```
Comment (rsrini7) [inline README.md:3]: instead of keeping 8pm hardcoded. can configure the time from env or configs

File: README.md [hunk 2]
```diff
@@ -1,41 +1,51 @@
...
-The script will continuously monitor the player list on the Aternos Minecraft server and update the Excel sheet whenever a new player joins or an existing player leaves the server. The Excel sheet will be updated with the following columns:
+The script will continuously monitor the player list on the Aternos Minecraft server and update the Excel sheet and CSV file whenever a new player joins or an existing player leaves the server. The Excel sheet and CSV file will be updated with the following columns:
 
 - Player name
 - Leave/Join
 - Time
 
+Additionally, the script will send a PDF file of the day's player list as an email attachment at 8:00 PM every day.
 
 ## Acknowledgements
```
Comment (rsrini7) [inline README.md:44]: Very good add Acknowledgements. :)

File: resources/.~lock.server-player-list.xlsx# [hunk 3]
```diff
@@ -0,0 +1 @@
+,MSI/Siddharth,MSI,05.05.2023 15:39,file:///C:/Users/Siddharth/AppData/Roaming/LibreOffice/4;
```
Comment (rsrini7) [inline resources/.~lock.server-player-list.xlsx#:1]: this should be temp file and should not be committed

**Review by rsrini7 — APPROVED** [review]

File: resources/.~lock.server-player-list.xlsx# (hunk 3 above)
Comment (siddvish21) [reply resources/.~lock.server-player-list.xlsx#:1]: Ohh

**Pull request merged**